homeassistant/components/climate/homematic.py

```diff
@@ -90,11 +90,11 @@ def target_temperature(self):
             return None
         return self._data.get('SET_TEMPERATURE', None)
 
-    def set_temperature(self, temperature):
+    def set_temperature(self, **data):
         """Set new target temperature."""
         if not self.available:
             return None
-        self._hmdevice.set_temperature(temperature)
+        self._hmdevice.set_temperature(data[ATTR_TEMPERATURE])
```

Comment:
:+1:

On Sep 5, 2016 11:56 AM, "Martin Hjelmare" <notifications@github.com> wrote:

> In homeassistant/components/climate/homematic.py
> <https://github.com/home-assistant/home-assistant/pull/3189#discussion_r77548112>
> :
>
> >          """Set new target temperature."""
> >          if not self.available:
> >              return None
> > -        self._hmdevice.set_temperature(temperature)
> > +        self._hmdevice.set_temperature(data[ATTR_TEMPERATURE])
>
> temp = data.get(ATTR_TEMPERATURE)
> if temp is None:
>     return
> self._hmdevice.set_temperature(temp)
>
> —
> You are receiving this because you commented.
> Reply to this email directly, view it on GitHub
> <https://github.com/home-assistant/home-assistant/pull/3189/files/c975a1158266846c89904a12a3b1630be316fb72#r77548112>,
> or mute the thread
> <https://github.com/notifications/unsubscribe-auth/AC2fZR9OxV9sZjHJVqtU2CgTNmZ1B0Z-ks5qnFfbgaJpZM4J0nMy>
> .
>